README.md

```diff
@@ -10,94 +10,94 @@ Pull requests are welcome. Please keep the list in alphabetical order.
-[Tyk](https://tyk.io/)                     | Cloud native API Management, open source                      | Go, Web, DevOps         | ✓                                  | [Jobs](https://tyk.io/about/team/)<br>[Glassdoor ](https://www.glassdoor.co.uk/Reviews/Tyk-Reviews-E2321465.htm)
-[Universal Mind](http://www.universalmind.com/)  | Digital agency              | Mobile, Web                | ✓                                  | [Jobs](http://www.universalmind.com/aboutus)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Universal-Mind-EI_IE648227.11,25.htm)
-[Wikimedia](https://wikimediafoundation.org/)    | Wikipedia                   | Web, PHP                   | ✗                                 | [Jobs](https://wikimediafoundation.org/wiki/Work_with_us)<br>[Glassdoor](https://www.glassdoor.com/Overview/Working-at-Wikimedia-Foundation-EI_IE38331.11,31.htm)
-[Xapo](https://xapo.com/)                    | Financial Technology   | Mobile, Web, Python                | ✓                                  | [Jobs](https://xapo.com/careers/)<br>[Glassdoor](https://www.glassdoor.com/Overview/Working-at-Xapo-EI_IE1843514.11,15.htm)
-[X-team](https://x-team.com/)                    | Remote full-time projects   | Mobile, Web                | ✓                                  | [Jobs](https://x-team.com/remote-programming-jobs/)<br>[Glassdoor](https://www.glassdoor.com/Overview/Working-at-X-Team-International-EI_IE1274500.11,31.htm)
-[Zapier](https://zapier.com/)                    | App connectors              | Python, Node, Web          | ✓                                  | [Jobs](https://zapier.com/jobs/)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Zapier-EI_IE1196705.11,17.htm)
+Company                                           | Business/products                        | Tech stack/skills                              | Globally-competitive compensation?   | Hiring From             | Job page & other links
+------------------------------------------------- | -----------------------------            | ----------------------------                   | ------------------------------------ | ----------------------- | ----------------------
+[10up](https://10up.com/)                         | Web consulting                           | WordPress, PHP, Node, Web                      | ✗                                    | ?                       | [Jobs](https://10up.com/careers/)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-10Up-EI_IE775906.11,15.htm)
+[15Five](https://www.15five.com)                  | Continuous performance management suite  | Django, Python, Javascript, React, Vue         | ✓                                    | US                      | [Jobs](https://www.15five.com/careers)<br>[Glassdoor](https://www.glassdoor.com/Overview/Working-at-15Five-EI_IE1391015.11,17.htm)
```

Comment:
Interesting that it's US-only. I may end up removing single-country companies since the list is growing and I don't see such companies as being truly distributed.